File: app/src/main/java/ru/samtakoy/core/data/local/database/room/entities/other/LearnCourseHelper.java
package ru.samtakoy.core.data.local.database.room.entities.other;

import java.util.List;

import ru.samtakoy.core.data.local.database.room.entities.LearnCourseEntity;


public class LearnCourseHelper {

    public static Long[] getLearnCourseIds(List<LearnCourseEntity> list) {
        Long[] result = new Long[list.size()];
        int i = 0;
        for (LearnCourseEntity learnCourse : list) {
            result[i++] = learnCourse.getId();
        }
        return result;
    }

}

File: module/data/common/impl/src/main/kotlin/ru/samtakoy/data/learncourse/utils/LearnCourseHelper.kt
package ru.samtakoy.data.learncourse.utils

import ru.samtakoy.domain.learncourse.LearnCourse

object LearnCourseHelper {
    fun getLearnCourseIds(list: List<LearnCourse>): Array<Long> {
        return list.map { it.id }.toTypedArray()
    }
}